### DISCUSSION TITLE
Weekly meeting notes

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#289] [General]
Kate, Lammert, I've set up a google [doc to keep track of our meetings and agreements](https://docs.google.com/document/d/1M1XlXMF5OmiduGdWuezLz0965L7mP4Wc). 

I'll be adding the items to the agenda throughout the week and then, after the meeting, summarizing the notes and adding actions for ourselves and the rest of the team. 

I've given you editor access. Feel free to add whatever you need to the agendas or notes!

**Comment by lyubovpencheva** [comment]
This is an old one.
